custom/icds/rules/custom_actions.py

```diff
@@ -113,7 +114,7 @@ def escalate_tech_issue(case, rule):
     tech_issue_delegate = _get_escalated_tech_issue_delegate(case, current_location_id)
 
     if tech_issue_delegate:
-        delegate_update_result = _update_existing_tech_issue_delegate(tech_issue_delegate)
+        delegate_update_result = _update_existing_tech_issue_delegate(tech_issue_delegate, escalated_location_id)
```

Comment:
Owner needs to be updated because when the case changes, there's a conditional alert set up to email the owner about their new issue.